### PR TITLE
Add a command to clear Local Storage

### PR DESCRIPTION
**Pull request by olegrise**
## Description
Add `clear-local-storage-request` command handler. See [design doc](https://docs.google.com/document/d/1v1-IJMKvSqSQ3picvYycLpZwYAQIJqO8TQb5_vwUkHA/edit?usp=sharing).

## Motivation and Context
This improvement was requested in Development Roadmap. It should also help with resolving the "stale content" issues (see [#832](https://github.com/Rise-Vision/rise-launcher-electron/issues/832) and [#829](https://github.com/Rise-Vision/rise-launcher-electron/issues/829)) if they ever occur.

Related PR: https://github.com/Rise-Vision/messaging-service/pull/83

## How Has This Been Tested?
Tested manually on stage.

## Release Plan:
- As the Submitter, upon requesting review of this pull request, I confirm that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed. 
- As the Reviewer, upon approving the changes in this PR, I confirm I have reviewed and I agree that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed

#### Release Checklist Items Skipped?
If any Release Checklist items were intentionally skipped, please provide which ones and the reasons why

### CODE REVIEW

File: src/messaging/clear-local-storage-request.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+module.exports = {
+  init() {
+    messagingServiceClient.on('clear-local-storage-request', ()=>{
+
+      logger.log('clear-local-storage-request');
+
+      db.fileMetadata.clear();
+      db.watchlist.clear();
+
```
Comment (andrecardoso) [inline src/messaging/clear-local-storage-request.js]: Maybe add a log here with the new state of the database just for debugging purposes, something like:

`logger.log('storage - database after clearing local storage', db.getEntireDBObject());`

File: src/messaging/clear-local-storage-request.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
+const db = require("../storage/database/api");
+const logger = require("../logging/logger");
+const messagingServiceClient = require('./messaging-service-client');
+const rebootScheduler = require('../reboot-scheduler');
+
+module.exports = {
+  init() {
+    messagingServiceClient.on('clear-local-storage-request', ()=>{
+
+      logger.log('clear-local-storage-request');
```
Comment (andrecardoso) [inline src/messaging/clear-local-storage-request.js]: Please add the prefix 'storage - ' to the log message

**Comment by olegrise** [comment]
@andrecardoso please review. Thanks.

**Comment by olegrise** [comment]
@andrecardoso just made a small change. I had it capitalized originally, but then it got overwritten somehow. Both ways work, so this change is just to be consistent with the rest of the code.

**Pull request merged**